PharoObjectModel/PharoObjectModel.pier

```diff
@@ -219,46 +219,50 @@ by instances of that class (and of its subclasses).
-
-For those familiar with Java and C\+\+,  class methods may seem similar to
-static methods. However, the uniformity of Pharo means that they are somewhat
-different: whereas Java static methods are really just statically-resolved
-procedures, Smalltalk class methods are dynamically-dispatched methods. This
-means that inheritance, overriding and super-sends work for class methods in
-Smalltalk, whereas they don't work for static methods in Java.
+% Commenting out for the moment: showColorCube is no longer in the image.
+% TODO: create a different example
+%Browse method ==Color class>>showColorCube==, double-click just inside the
+%quotes on the comment =="Color showColorCube"== and press ==CMD-d==. You will
+%see the effect of executing this method. Select
+%==World > System > Restore display (r)== to undo the effects.
+
+For those familiar with Java and C\+\+, class methods may seem similar to static
+methods. However, the uniformity of the Pharo object model (where classes are
+just regular objects) means that they are somewhat different: whereas Java
+static methods are really just statically-resolved procedures, Pharo class
+methods are dynamically-dispatched methods. This means that inheritance,
+overriding and super-sends work for class methods in Pharo, whereas they don't
+work for static methods in Java.
 
 !!!Class instance variables
 
 With ordinary instance variables, all the instances of a class have the same set
-of variable names, and the instances of its subclasses inherit those names;
-however, each instance has its own private set of values.The story is exactly
-the same with class instance variables: each class has its own private class
-instance variables.A subclass will inherit those class instance variables, ''but
-it has its own private copies of those variables''. Just as objects don't share
-instance variables, neither do classes and their subclasses share class instance
-variables.
+of variable names (though each instance has its own private set of values), and
+the instances of its subclasses inherit those names. The story is exactly the
+same with class instance variables: each class has its own private class
+instance variables. A subclass will inherit those class instance variables,
+''but it has its own private copies of those variables''. Just as objects don't
+share instance variables, neither do classes and their subclasses share class
+instance variables.
 
-You could use a class instance variable called ==count== to keep track of how
-many instances you create of a given class. However, any subclass would have its
-own ==count== variable, so subclass instances would be counted separately.
+For example, you could use a class instance variable called ==count== to keep
+track of how many instances you create of a given class. However, any subclass
+would have its own ==count== variable, so subclass instances would be counted
+separately.
 
-""Example: class instance variables are not shared with subclasses.""
```

Comment:
I wasn't removing the line, was just changing it to use a minor level heading (`!!!!`) instead of italics. (To match the formatting of the other examples).